doc/releases/changelog-0.11.0.md

```diff
@@ -202,78 +217,81 @@
   for the `StatePrep`, `BasisState`, and `QubitUnitary` operations. For the parameter-shift method,
   this is allowable for all operations.
 
-* Changes to support a dynamic number of qubits:
+* Several changes have been made to support dynamic qubit specification:
```

Comment:
This is the missing feature @josh146 is looking for, let's move it to the top.
```suggestion
* Catalyst now supports qubit number-invariant compilation! That is, programs can be compiled without
  specifying the number of qubits to allocate ahead of times. Instead, the device can be supplied with
  a dynamic program variable as the number of wires.
```

We also need to add an example to this feature.